wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -69,3 +70,20 @@ def visit_any_function(self, node: AnyFunctionDef) -> None:
         self._check_decorators(node)
         self._check_magic_methods(node)
         self.generic_visit(node)
+
+
+class WrongParentClassListVisitor(BaseNodeVisitor):
+    """Check parent class list."""
+
+    def visit_ClassDef(self, node: ast.ClassDef) -> None:
+        """Check 'object' class in parent list."""
+        if len(node.bases) > 1:
+            for base_name in node.bases:
+                id_attr = getattr(base_name, 'id')
+                if id_attr is not None:
+                    self._check_node_name(base_name)
```

Comment:
You are missing `self.generic_visit(node)` line here (in the end of `visit_ClassDef`)